thirdparty/wasi/api.hpp

```diff
@@ -1715,6 +1715,7 @@ static_assert(offsetof(__wasi_address_t, buf_len) == 4,
 
 enum __wasi_sock_opt_level_t : uint32_t {
   __WASI_SOCK_OPT_LEVEL_SOL_SOCKET = 0,
+  __WASI_SOCK_OPT_LEVEL_IPPROTO_TCP = 1,
```

Comment:
I think we can use `6` instead of `1`. Follow the https://github.com/leostratus/netinet/blob/master/in.h#L43

Reply:
yes, pretty much what I was wondering. Was not sure WasmEdge was following the same number assignment. Will change to 6! thanks